README.md

```diff
@@ -289,6 +289,7 @@ If you want to add a new loader, for example, to process `json` files via webpac
 
 ```
 yarn add json-loader
+npm install --save json-loader
```

Comment:
npm >=5 will save by default, so `--save` is not required (4th bullet on the [changelog for 5.0.0](https://github.com/npm/npm/releases/tag/v5.0.0))